lms/envs/common.py

```diff
@@ -35,6 +35,14 @@
 from corsheaders.defaults import default_headers as corsheaders_default_headers
 from path import Path as path
 from django.utils.translation import ugettext_lazy as _
+from enterprise.constants import (
+    ENTERPRISE_LEARNER_ROLE,
```

Comment:
We are not using this role in the mapping ... I guess this import should be removed.

Reply:
Done